lib/exercises.js

```diff
@@ -1,17 +1,55 @@
 // This method will return an array of arrays.
 // Each subarray will have strings which are anagrams of each other
-// Time Complexity: ?
-// Space Complexity: ?
+// Time Complexity: O(n+m)?
+// Space Complexity: O(n)
 function grouped_anagrams(strings) {
```

Comment:
Since you are sorting the solution is O(n * m log m) where n is the length of the list and m is the biggest number of characters in the individual strings.